Sources/VLCActionSheet/VLCActionSheet.swift

```diff
@@ -200,6 +200,12 @@ class VLCActionSheet: UIViewController {
         // This is to avoid a horrible visual glitch!
         mainStackView.isHidden = false
 
+        let roundedCornerPath = UIBezierPath(roundedRect: headerView.bounds, byRoundingCorners: [.topLeft, .topRight],
+                                             cornerRadii: CGSize(width: 10, height: 10))
+        let maskLayer = CAShapeLayer()
+        maskLayer.path = roundedCornerPath.cgPath
+        headerView.layer.mask = maskLayer
```

Comment:
you can just set the cornerRadius of the views layer

Reply:
Just tried it but didn't work :(

The actionSheet is `headerView` + `stackView` and so I think the corner radius of the headerView must be set.

Reply:
you also need to set maskstobounds I think

Reply:
Yep tried that too 🤭

Reply:
Ah I should have been clearer! Sorry about that

Reply:
yeah it worked with just setting the cornerradius of the headerview but it obviously didn't have the wanted effect. I'm actually curious how the Maps app build it

Reply:
Hmmm good question. I don’t think they are using a stack view for the sheet unlike us. Maybe just a table view with the table’s corner radius set??